### ISSUE TITLE
Print button on Reports missing

### ISSUE THREAD

**Issue by AceLeader2023** [pete-mc/Summit#19]
Just comparing what is on my screen to what is on the screenshots on Facebook. I currently have no print button. 

Is this something that could get added if it is not already added??

I have attached screenshots below of what I am getting on Chrome.
<img width="1028" alt="image" src="https://github.com/pete-mc/Summit/assets/110509017/f017686c-4607-4f4e-926a-e4873b5d994e">
<img width="1021" alt="image" src="https://github.com/pete-mc/Summit/assets/110509017/c2f783f5-8df6-4c2f-b09d-0bcf577294c6">

Let me know if you need more information.

**Comment by AceLeader2023** [comment]
Can print normally, with ctrl + P. 

All good.

**Comment by pete-mc** [comment]
You're right I think there is an issue with the pdf and Excel maker. I will reopen this to try and fix. I think it happened when I reworked how alot of the code interconnects.

**Comment by pete-mc** [comment]
Completed - added buttons back in and also cleaned up formatting on tables and made them all a bit cleaner. scheduled for release in 2.3.0.
